Funqy Google Cloud Functions codestart

loicmathieu opened the pull request.

loicmathieu committed:
Remove unused classes from Funqy GCF IT

loicmathieu committed:
Add a codestart for Funqy Google Cloud Functions

loicmathieu commented:
/cc @ia3andy the last codestart for Google Cloud Functions that gives example on the third way to build a function: via Funqy.

ia3andy commented on devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/base/README.md:
```diff
@@ -0,0 +1,10 @@
+
+Guide: https://quarkus.io/guides/funqy-gcp-functions
+
+Two examples have been generated under `src/main/java/org/acme/fuqygooglecloudfunctions`, you must remove them before deploying to 
```
```suggestion
Two examples have been generated under `src/main/java/org/acme/funqygooglecloudfunctions`, you must remove them before deploying to 
```

loicmathieu replied:
Nice catch, fixed!

ia3andy reviewed (changes requested):
Cool!

gsmet commented:
@ia3andy could you have another look?

ia3andy commented:
I added a new `metadata` section in the codestart.yml, and added a template for the readme. I will push a commit today to update this PR with it..

loicmathieu commented:
> I added a new metadata section in the codestart.yml, and added a template for the readme. I will push a commit today to update this PR with it..

@ia3andy do you have time to do it today . 1.12 will be cut off tomorrow I would like to have this one included inside it

ia3andy commented:
Ok! I will :)

ia3andy commented:
@loicmathieu could you rebase the PR from master please?

ia3andy commented:
I created a new PR from yours https://github.com/quarkusio/quarkus/pull/14942
